src/index.tsx

```diff
@@ -101,6 +102,10 @@ window.__APP_ANALYTICS = registerGlobalAnalytics(window, app.store);
 pollUpdates(app.store);
 startMathJax();
 
+// load optimize
+loadOptimize(config.DEPLOYED_ENV === 'server' ? null
+  : (window.location.hostname === 'openstax.org' ? 'OPT-NFHSM4B' : 'OPT-W65B3CP'));
```

Comment:
so in this file we know we're in a browser, pre-rendering doesn't use this file 

i'd do like... `getOptimizeContainerByEnv` or something within your `src/helpers/loadOptimize.ts` file and make this here just a one like function call with no conditions

Reply:
whoops, yeah, forgot to make the switch when I changed this from `APP_ENV` to `DEPLOYED_ENV`